### ISSUE TITLE
Login issue

### ISSUE THREAD

**Issue by DanielLaszczych** [RoboTigers/FRCScout2020#36]
I found a bug that doesn't happen locally but does on Heroku. Basically when you go to the login page and then you reload (reload as in F5) a different screen shows up, seems like a screen related to the backend. Then when you log in with a valid user on this screen it shows a JSON of the username and role. And when you log in with an invalid user it shows 'unauthorized'. Here are some screenshots: 

![image](https://user-images.githubusercontent.com/14986272/74743716-e4dec780-522e-11ea-976e-a2a50f4c0ba8.png)

Then I reload:

![image](https://user-images.githubusercontent.com/14986272/74743345-38044a80-522e-11ea-8ddd-50de055f8652.png)
If valid:
![image](https://user-images.githubusercontent.com/14986272/74743373-4488a300-522e-11ea-9ae2-e3fc2b4eff69.png)
If invalid:
![image](https://user-images.githubusercontent.com/14986272/74743404-523e2880-522e-11ea-95f8-51ca2e1fdfe4.png)

Basically, it seems like it's only showing the backend. Here are some logs that may or may not help:

![image](https://user-images.githubusercontent.com/14986272/74743490-7994f580-522e-11ea-9718-c3785ee1bb1e.png)
![image](https://user-images.githubusercontent.com/14986272/74743287-1e630300-522e-11ea-949e-b790c65a628c.png)

You can reproduce the bug https://daniel-scout-2020.herokuapp.com/
I made an account with the username: Matt and password: password if you want to test things out.

_Originally posted by @DanielLaszczych in https://github.com/RoboTigers/FRCScout2020/pull/35#issuecomment-587481919_

**Comment by MGerrior** [comment]
This should be resolved by #37
